runtime/lib/ttnn/program.cpp

```diff
@@ -92,6 +92,14 @@ run(::tt::target::ttnn::EltwiseOp const *op, ::ttnn::Device &device,
     liveTensors.try_emplace(op->out()->global_id(), &tensorPool.back());
     break;
   }
+  case ::tt::target::ttnn::EltwiseOpType::GreaterEqual: {
+    assert(op->ins()->size() == 2 && "Unsupported number of inputs");
+    auto &lhs = *liveTensors.at(op->ins()->Get(0)->global_id());
+    auto &rhs = *liveTensors.at(op->ins()->Get(1)->global_id());
```

Comment:
We prefer correctly naming types instead of auto unless it's some iterator or an untrivial type.

Reply:
Thanks for the comment. updated.